Controller/Payment/Order.php

```diff
@@ -42,10 +43,91 @@ public function __construct(
         $this->catalogSession  = $catalogSession;
         $this->config          = $config;
         $this->logger          = $logger;
+        $this->webhookId       = null;
+        $this->_storeManager   = $storeManager;
+        $this->webhookUrl      = $this->_storeManager
+                                    ->getStore()
+                                    ->getBaseUrl(\Magento\Framework\UrlInterface::URL_TYPE_WEB) .
+                                    'razorpay/payment/webhook';
     }
 
     public function execute()
     {
+        if(empty($this->config->getConfigData('webhook_triggered_at')) === false)
+        {
+            $webhookUpdatedAt = (int) $this->config->getConfigData('webhook_triggered_at');
+            $diffrenceInHour = abs(time() - $webhookUpdatedAt)/(60*60);
```

Comment:
@enishant , in place of calculating hours. just use current time stamp and add 24 hours then validate it.

Reply:
@ramth05 Added 24 hours to webhookTriggeredAt timestamp and validated with current timestamp.